Author: Pedro Avalos Jimenez <pedro.avalosjimenez@canonical.com>
Description: add install target

--- a/Makefile
+++ b/Makefile
@@ -41,6 +41,18 @@ FILTER_OUT :=
 
 PROJECTS := $(filter-out $(FILTER_OUT),$(PROJECTS))
 
+ifeq ($(PREFIX),)
+    PREFIX := /usr/local
+endif
+
+.PHONY: all
+all: build
+	@echo "Finished building CUDA samples"
+
+.PHONY: install
+install:
+	find bin/$(TARGET_ARCH)/linux/release -type f -exec install -D "{}" "$(DESTDIR)$(PREFIX)/cuda/samples/bin/{}" \;
+
 %.ph_build :
 	+@$(MAKE) -C $(dir $*) $(MAKECMDGOALS)
 
@@ -53,9 +65,6 @@ PROJECTS := $(filter-out $(FILTER_OUT),$(PROJECTS))
 %.ph_clobber :
 	+@$(MAKE) -C $(dir $*) clobber $(USE_DEVICE)
 
-all:  $(addsuffix .ph_build,$(PROJECTS))
-	@echo "Finished building CUDA samples"
-
 build: $(addsuffix .ph_build,$(PROJECTS))
 
 test : $(addsuffix .ph_test,$(PROJECTS))
